docs/ui-notes.md

```diff
@@ -0,0 +1,40 @@
+# UI Notes
+
+### Miscellaneous
+- toolbar background: `<Sheet variant='soft'>` or `<Sheet className='TapSearch__section' variant='outline' sx={{flexDirection: 'column', flexGrow: 1}}>`
+
+### Buttons
+- primary button:  `<Button size:'md', variant: 'solid' color: 'primary'/>`
+- secondary button:  `<Button/>` (see above for default)
+- significant action button: (eg file upload): `<Button color='success' variant='solid'/>`
+- other buttons: `<Chip/>` (see above for defaults)
+
+### Text
+- information: `<Typography/>`
+- labels: `<Typography/>`
+- feedback: `<Typography color='warning'/>`
```

Comment:
Good point.  I wonder if we should further define level as well. e.g.
```js
label: <Typography level='title-x'/>
information: <Typography level='body-x'/>
```

Reply:
I just copy pasted it from what was present at ThemeSetup. Also I don't see any level named like these, did you mean 'title-sm', 'body-xs'?

Please feel free to edit it in your upcoming PRs

Reply:
I mean when text is `body-sm`, then label for that text should be `title-sm`.
Basically, use `title` for label, and `body` for normal text.  They will produce consistent padding.

Reply:
Oh I see. Yes, that will be helpful!